prebid-mobile/adops-line-item-setup-mopub.md

```diff
@@ -11,12 +11,12 @@ sidebarType: 3
 
 
 
-# Step by Step Line Item Setup for MoPub
+# Step-by-Step Line Item Setup for MoPub
 
 * TOC
 {:toc }
 
-This page describes step by step how to set up Prebid Mobile line items for MoPub to serve ads on app with the Prebid SDK. It is using the Universal Prebid Creative.
+This page describes step-by-step instructions to set up Prebid Mobile line items for MoPub to serve ads on app with the Prebid SDK. It is using the Universal Prebid Creative.
```

Comment:
We're not actually describing the instructions, the instructions _are_ the description. Would be better to say "This page provides step-by-step instructions..."